test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -401,65 +442,99 @@ export default class PerfCompare extends Component {
+                    for ( let y = 0; y < this.state.benchmarkRuns.childBuildsListTest.length; y++ ) {
+                        childBuildTestData["child " + (y + 1) ] = this.state.benchmarkRuns.childBuildsListTest[y][j].value;
+                    }
                 }
 
-                try {
-                    curTestCI = math.round((this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].ci), 3);
-                    curTestCI += "%";
-                } catch(e) {
-                    curTestCI = "Not found";
+                // raw values displaying: master jobs diplay all their children raw values, and child jobs diplay their own raw values.
+                if ( this.state.buildType === "Jenkins") {
+                    curRawValues = {
+                        "baseline": {
+                            "Overview": parsedVariantsBaseline.metrics[j].value,
+                            "Raw Data": (parsedVariantsBaseline.testsData === undefined) ? childBuildBaselineData : this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].testsData[j].value
+                         },
+                        "test": {
+                            "Overview": parsedVariantsTest.metrics[curMatchingTestMetricIndex].value,
+                            "Raw Data": (parsedVariantsTest.testsData === undefined) ? childBuildTestData : parsedVariantsTest.testsData[j].value
+                         }
+                    };
+
+                    try {
+                        curBaselineScore = math.round((parsedVariantsBaseline.metrics[j].value.mean), 3);
+                        curTestScore = math.round((parsedVariantsTest.metrics[curMatchingTestMetricIndex].value.mean), 3);
+                        curBaselineCI = math.round((parsedVariantsBaseline.metrics[j].value.CI), 3);
+                        curBaselineCI += "%";
+                        curTestCI = math.round((parsedVariantsTest.metrics[curMatchingTestMetricIndex].value.CI), 3);
+                        curTestCI += "%";
+                    } catch (e) {
+                        curBaselineScore = "error";
+                        curTestScore = "error";
+                        curBaselineCI = "Not found";
+                        curTestCI = "Not found";
+    
+                    }
+                } else {
+                    // perffarm builds
+                    curRawValues = {
+                        "baseline": this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].value,
+                        "test": this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].value
+                    };
+
+                    try {
+                        curBaselineScore = math.round((this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].metrics[j].mean), 3);
```

Comment:
This code could be factored to be used for both Perffarm and Jenkins. We'll need to change these lines
https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/7bd24e101fa667ea601e8094be4c50277e60bc18/test-result-summary-client/src/PerfCompare/lib/PerffarmRunJSON.js#L84-L87
from `ci` to `CI`.